misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -15,6 +15,7 @@ data class WebConfig(
   val queue_size: Int? = null,
   val jetty_max_thread_pool_size: Int? = null,
   val action_exception_log_level: ActionExceptionLogLevelConfig = ActionExceptionLogLevelConfig(),
+  val jetty_max_concurrent_streams: Int? = null,
```

Comment:
tempted to rename this to `max_concurrent_streams_per_connection` since it’s not the max streams overall

Reply:
(is it?)

Reply:
I'll clean up this and other comment in a follow up PR